devel/hs-transformers/Portfile

```diff
@@ -0,0 +1,20 @@
+PortSystem          1.0
+PortGroup           haskell 1.0
+
+haskell.setup       transformers 0.4.3.0
```

Comment:
Isn't hs-transformers already part of GHC? I can't imagine we wouldn't already have such a fundamental package.